src/libs/blueprint/python/conduit_blueprint_mesh_examples_python.cpp

```diff
@@ -630,6 +630,7 @@ PyBlueprint_mesh_examples_polytess(PyObject *, //self
     Node &node = *PyConduit_Node_Get_Node_Ptr(py_node);
 
     blueprint::mesh::examples::polytess(nlevels,
+                                        1,
```

Comment:
I'll need some guidance @cyrush  on how to get this working... it'd be nice if people using python weren't limited to 2 dimensions for their polytessalations.